R/readtxt.R

```diff
@@ -163,7 +163,17 @@ getSource <- function(f, textfield, ...) {
     ## SIMPLER -KB
     fileType <- file_ext(f)
     if (!(fileType %in% SUPPORTED_FILETYPE_MAPPING))
-        stop(paste('Unsupported extension', fileType, 'of file', f))
+        if (dir.exists(f)) {
+            call <- deparse(sys.call(1))
+            call <- sub(f, paste0(sub('/$', '', f), '/*'), call, fixed=T)
```

Comment:
Here's a good illustration of why to use `TRUE` instead of `T`, because if you had used an uppercase `F` in the function signature instead of `f` then it would have overwritten the mapping of `F` to `FALSE`.

`T` and `F` should *never* have been valid substitutes for `TRUE` and `FALSE` in R...